includes/lom-capture.php

```diff
@@ -83,6 +83,12 @@ function lom_capture_ledyer_order( $order_id, $api, $action = false ) {
 		return;
 	}
 
+	if ( ! $order->get_meta( '_ready_for_capture', true ) ) {
+		$order->add_order_note( 'Ledyer order is not ready for capture' );
+		$order->save();
+		return;
+	}
+
```

Comment:
This leaves the order as still showing to be captured in WooCommerce right? Should we set the order to on-hold here to show the merchant that something went wrong?

Reply:
@anyakro Lets discuss this internally before we make any changes